charts/opensearch-operator/templates/opensearch-operator-controller-manager-deployment.yaml

```
@@ -53,6 +53,10 @@ spec:
         livenessProbe:
 {{- toYaml .Values.manager.livenessProbe | nindent 10 }}
         env:
+        {{- if .Values.UpgradeChecker.env }}
+        - name: UpgradeChecker
+          value: "{{ .Values.UpgradeChecker.env }}"
```

Helm suggests using the `quote` helper instead of using actual quotes
```suggestion
          value: {{ .Values.UpgradeChecker.env | quote }}
```